docs/README.md

```diff
@@ -58,6 +58,7 @@
 ## [Features and components](06-features-and-components/)
 - [SteadyPage JS utility](06-features-and-components/01-steadypage-js-utility.md)
 - [How are trail pictures picked in Frontend?](06-features-and-components/02-trail-images.md)
+- [theguardian.com/font-loader](06-features-and-components/03-font-loader-route.md)
```

Comment:
Is this meant to be part of this PR?

Reply:
Hmm, not sure why it is here. I can see it on `main` though so not too concerned. I'll rebase in any case which might disappear it.